### PR TITLE
Selecting archive type for the release depending on version

### PR DESCRIPTION
**Pull request by LDaneliukas**
The releases used to use `gz` up until version `v1.1.0`, which is when all new releases are coming in `zip`.

### CODE REVIEW

File: manifests/install.pp [hunk 1]
```diff
@@ -16,14 +16,20 @@
 
   $version_dir = "${data_dir}/promtail-${promtail::version}"
   $binary_path = "${version_dir}/${release_file_name}"
+  
+  if versioncmp($promtail::version, 'v1.0.0') > 0 {
```
Comment (genebean) [inline manifests/install.pp]:
```suggestion
  if versioncmp($promtail::version, 'v1.0.0') >= 0 {
```

I validated this by running the following locally with `puppet apply`:

```puppet
$versions = [
  'v0.3.0',
  'v1.0.0',
  'v1.0.1',
]

$versions.each |$v| {
  if versioncmp($v, 'v1.0.0') >= 0 {
    notice("${v} >= v1.0.0, use zip")
  } else {
    notice("${v} < v1.0.0, use gz")
  }
}
```

Comment (genebean) [reply manifests/install.pp]: Also @LDaneliukas - please squash your changes into a single commit.

Comment (LDaneliukas) [reply manifests/install.pp]: I've done `>` as `v1.0.0` is using `.gz`. we can do `>=` and adjust the version to `v1.1.0` in the comparison.

I'm away now but I can update this tomorrow.

Comment (genebean) [reply manifests/install.pp]: My bad, `>` would be correct then. All that’s needed now is lint fixes and squashing

**Pull request closed without merging**